webpack/webpack.config.js

```diff
@@ -237,36 +223,12 @@ module.exports = function(env, { /* @if jasmine || mocha*/runTest, /* @endif */a
               defaultShadowOptions: { mode: 'open' }
             }
           },
-          exclude: /node_modules/
-        }
-        // @endif
-        // @if css-module
```

Comment:
we dont need this anymore?

Reply:
No. css-module now follows the foo.module.css convention. Webpack css-loader supports it by default, no more explicit options to the loader.

Reply:
Note `exclude: /node_modules/` is not removed, it's on line 267. That's the funny way git shows the diff.